src/m365/teams/commands/chat/chat-member-list.ts

```diff
@@ -0,0 +1,63 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import Utils from '../../../../Utils';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  chatId: string;
+}
+
+class TeamsChatMemberListCommand extends GraphItemsListCommand<any> {
+  public get name(): string {
+    return commands.CHAT_MEMBER_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all members from a chat';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['userId', 'displayName', 'tenantId'];
```

Comment:
Rather than showing the `tenantId` in the default overview, what if we showed user's email? Wouldn't email be more useful for a quick lookup or is there a specific reason that you included the tenantId instead?

Reply:
You're right, don't know what I was thinking here. 👍